sync_diff_inspector/diff.go

```diff
@@ -423,6 +431,11 @@ func generateDML(tp string, data map[string][]byte, keys []*model.ColumnInfo, ta
 	case "delete":
 		kvs := make([]string, 0, len(keys))
 		for _, col := range keys {
+			if null[col.Name.O] {
+				kvs = append(kvs, fmt.Sprintf("%s is NULL", col.Name.O))
```

Comment:
Should we use backquote for column name?

Reply:
done